# Give pose
geometry_msgs/Point world_point

---

float64 cost